packages/core/test/pseudo-states.spec.ts

```diff
@@ -2027,6 +2027,34 @@ describe('pseudo-states', () => {
                 });
             });
         });
+
+        describe('state inside selector list', () => {
```

Comment:
I know I said that  maybe `pseudo-states.spec` is a good place for this, but I mislead you. this spec is for stylable custom states. this is simply a transform of functional pseudo-classes. There is no specific spec file for pseudo-classes, so maybe add a describe for `pseudo-classes` in `scoping.spec` and place it under there for now.